WordPress/gutenberg
# Add the command center to the post editor

Related #48457

## What?

This PR adds the command center (cmd + k) to the post editor. While this command center is meant to be added to all WPAdmin pages, let's start with the React ones to test it properly.

## Testing Instructions

Make sure the command center experiment is enabled.

1- Open the post editor
2- Hit cmd+k
3- Use the command menu
